### ISSUE TITLE
Nest Protect Brightness Low go to level Medium.

### ISSUE THREAD

**Issue by alrooijen** [iMicknl/ha-nest-protect#262]
### The problem

Hi,

The Brightness select level low also goes to medium level in place of low. I check this with the Nest phone app. When I control in HA the brightness level to high I see this changed in the app also to high but when I change the level in HA to low or medium both it will go in the app to medium. It looks like the level low do the same command als the medium level.

### What version of this integration (ha-nest-protect) has the issue?

0.3.12

### What version of Home Assistant Core has the issue?

Home Assistant 2023.9.2; Supervisor 2023.09.2; Operating System 10.5 Frontend-versie: 20230911.0 - latest

### Device / Model

Nest Protect (Battery)

### Home Assistant log

<details><summary>Logs</summary>
```
Copy/paste any log here, between the starting and ending backticks (`)
```

</details>

**Comment by iMicknl** [comment]
@alrooijen can you share your diagnostics? Do you use the cookie method, or the old authentication method?

I tried to reproduce this, but I can't.

**Comment by alrooijen** [comment]
@iMicknl I use the cookie method. I found what was the issue. In HA when i choose high in the nest app I see also high. Medium works also fine. But when I choose Low in HA in the nest app it going to medium but when I want to select a different light level in the nest app phone I get a popup notification "if the change u made direcly active press the nest button" after I press OK in notitification it works fine. When I select low in HA I see this now also in the nest app. Only thing what I did is in the nest app press OK on the popup notification. I did not press the nest button.
